src/utils/application.data.ts

```diff
@@ -36,10 +35,10 @@ export const prepareData = (data: any, keys: string[]): ApplicationDataType => {
   return keys.reduce((o, key) => Object.assign(o, { [key]: data[key] }), {});
 };
 
-export const mapObjectFieldToAddress = (data: any, keys: string[]) => {
-  return keys.reduce((o, key, i) => Object.assign(o, { [dataType.AddressKeys[i]]: data[key] }), {});
+export const mapFieldsToDataObject = (data: any, keys: string[], mapKeys: string[]) => {
```

Comment:
we could name the keys parameter to be htmlFields and mapKeys to be dataModelKeys? I think that make it clearer what data we pass into the parameters
same for the mapDataObjectToFields function

Reply:
Thanks Mark, agreed